src/mlrose_ky/algorithms/decay/geometric_decay.py

```diff
@@ -18,75 +20,81 @@ class GeomDecay:
 
     Parameters
     ----------
-    init_temp : float, default=1.0
+    initial_temperature : float, default=1.0
```

Comment:
Why did method properties and attributes get renamed? This is a regression to the `mlrose-ky` naming conventions that we are trying to undo with #7; the correct names for these parameters, for example, are `init_temp`, `decay`, etc.